script/docker-entrypoint.sh

```diff
@@ -45,5 +45,5 @@ else
 fi
 
 pushd build
-GTEST_OUTPUT=xml:test-results ctest -V
+GTEST_OUTPUT=xml:$(pwd)/test-results ctest -V
```

Comment:
tests were getting dumped under the `engine` directory from what I can tell; this should enforce them being in where we expect them to be